src/python/pants/backend/python/tasks/setup_py.py

```diff
@@ -546,8 +551,8 @@ def convert(input):
         return out
       elif isinstance(input, list):
         return [convert(element) for element in input]
-      elif isinstance(input, string):
-        return to_bytes(input)
+      elif PY2 and isinstance(input, str):
```

Comment:
If Py3, keep as unicode

Reply:
Exactly, due to this:

> This file is a little hacky to get to work on both versions.
>
>It relies on pprint to generate the content written to file. In Py2, unicode is prefaced with u, and in Py3 bytes are prefaced with b. The goal is to have no prefix displayed, so to do this we have to use bytes in Py2 and unicode in Py3.

I hate us having to have different implementations between Py2 and Py3, but I'm not sure how else to get this to work.

Reply:
Ok. That kind of comment should probably go in the code.